libs/application/data-providers/src/providers/FamilyInformationProvider.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  BasicDataProvider,
+  SuccessfulDataProviderResult,
+  FailedDataProviderResult,
+} from '@island.is/application/core'
+
+import { NationalRegistryFamilyMember } from '@island.is/api/schema'
+
+export class FamilyInformationProvider extends BasicDataProvider {
+  type = 'FamilyInformationProvider'
+
+  async provide(): Promise<NationalRegistryFamilyMember[]> {
+    const query = `
+      query NationalRegistryFamilyQuery {
+        nationalRegistryFamily {
+          nationalId
+          fullName
+          familyRelation
+        }
+      }
+    `
+
+    return this.useGraphqlGateway(query)
+      .then(async (res: Response) => {
+        const response = await res.json()
+        if (response.errors) {
+          return this.handleError()
+        }
+
+        return Promise.resolve(response.data.nationalRegistryFamily)
+      })
+      .catch(() => {
+        return this.handleError()
+      })
+  }
+
+  handleError() {
+    return Promise.resolve([])
```

Comment:
No errors handlings?

Reply:
The `onProvideError` will get called with

```
{
  date: new Date(),
  reason: result,
  status: 'failure',
  data: result,
}
```

Where `result` is whatever handleErrors resolves to.

But I'll include logging when this happens.